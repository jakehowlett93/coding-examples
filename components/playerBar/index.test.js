import PlayerBar from './index';

describe('playerBar', () => {
	const state = {
		room: { players: [{ name: 'jake' }, { name: 'joe' }, { name: 'ben' }, { name: 'ant' }] }
	};
	let returnValue;
	beforeEach(() => {
		returnValue = PlayerBar({ state });
	});
	it('should write the right component ', () => {
		expect(returnValue).toMatchSnapshot();
	});
});
