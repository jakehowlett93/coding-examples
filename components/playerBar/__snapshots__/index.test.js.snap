// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`playerBar should write the right component  1`] = `
<div
  className="playerBar"
>
  <h1>
    Players: 
  </h1>
  <ul
    className="playerList"
  >
    <li>
      <p
        className="waiting"
      >
        jake
      </p>
    </li>
    <li>
      <p
        className="waiting"
      >
        joe
      </p>
    </li>
    <li>
      <p
        className="waiting"
      >
        ben
      </p>
    </li>
    <li>
      <p
        className="waiting"
      >
        ant
      </p>
    </li>
  </ul>
</div>
`;
